Compilation on CLI fails with below errors

**Affected module**
Does it impact the UI, backend or Ingestion Framework?

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project common: Compilation failure: Compilation failure: 
[ERROR] /Users/mohityadav/IdeaProjects/OMTesting/common/src/main/java/org/openmetadata/common/utils/CommonUtil.java:[88,5] cannot find symbol
[ERROR]   symbol:   variable LOG
[ERROR]   location: class org.openmetadata.common.utils.CommonUtil
[ERROR] /Users/mohityadav/IdeaProjects/OMTesting/common/src/main/java/org/openmetadata/common/utils/CommonUtil.java:[96,11] cannot find symbol
[ERROR]   symbol:   variable LOG
[ERROR]   location: class org.openmetadata.common.utils.CommonUtil
[ERROR] /Users/mohityadav/IdeaProjects/OMTesting/common/src/main/java/org/openmetadata/common/utils/CommonUtil.java:[115,17] cannot find symbol
[ERROR]   symbol:   variable LOG
[ERROR]   location: class org.openmetadata.common.utils.CommonUtil
[ERROR] /Users/mohityadav/IdeaProjects/OMTesting/common/src/main/java/org/openmetadata/common/utils/CommonUtil.java:[242,7] cannot find symbol
[ERROR]   symbol:   variable LOG
[ERROR]   location: class org.openmetadata.common.utils.CommonUtil
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.


**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**

Screenshots or steps to reproduce

**Expected behavior**
A clear and concise description of what you expected to happen.

**Version:**
 - OS: [e.g. iOS]
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**
Add any other context about the problem here.

--- a/pom.xml
+++ b/pom.xml
@@ -884,6 +884,13 @@
           <configuration>
             <source>17</source>
             <target>17</target>
+            <annotationProcessorPaths>
+              <path>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok</artifactId>
+                <version>${lombok.version}</version>
+              </path>
+            </annotationProcessorPaths>
           </configuration>
         </plugin>
         <plugin>

